recipes/pylhe/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pylhe" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pylhe-{{ version }}.tar.gz
+  sha256: 75d179d8abf21ea2686bb25d9e86e2f3c91cce5c251c3deb267ee6ac46c7e439
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - awkward >=1.2.0
+    - python-graphviz >=0.12.0
+    - networkx >=2.2
+    - particle >=0.16
+    - python >=3.6
+    - tex2pix >=0.3
```

Comment:
`tex2pix` isn't on Conda-forge and I guess this is problematic, so I'll need to figure out how to tell Conda-forge about the sdist on https://pypi.org/project/tex2pix/.

Reply:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#avoid-external-dependencies:

> As a general rule: all dependencies have to be packaged by conda-forge as well. This is necessary to assure [ABI](https://conda-forge.org/docs/misc/00_intro.html#term-ABI) compatibility for all our packages.

So either need to package `tex2pix` or address https://github.com/scikit-hep/pylhe/issues/53.

Reply:
Personally I would rather deal with https://github.com/scikit-hep/pylhe/issues/53. The making of a PDF from a figure is indeed something very decoupled from `pylhe`'s goals and scope, and we should avoid unnecessary (package) couplings. The production of a figure is something easy following https://github.com/scikit-hep/pylhe/pull/118, anyway, and it is now more a matter of showing an example of how to produce a PNG or PDF, see for example https://graphviz.readthedocs.io/en/stable/api.html?highlight=pdf#id3.

I can probably help on these matters in the next few days ... Let me know.

Reply:
See https://github.com/scikit-hep/pylhe/pull/131.

Reply:
MR https://github.com/scikit-hep/pylhe/pull/131 is now merged. I will remove the 2 obsoleted dependencies here. Of course the finalisation of this MR will require a release of `pylhe`, to be done soon.